node-tests/test.js

```diff
@@ -1,15 +1,16 @@
+'use strict'
+
 var fs = require('fs-extra');
 var exec = require('child_process').exec;
 
-var expect = require('chai').expect;
+var chai = require('chai');
+chai.use(require('chai-things'));
+var expect = chai.expect;
 
 var FAILING_FILE = __dirname + '/../tests/dummy/app/unused.js';
 
-var execSync = require('child_process').execSync;
-
-var chromeVersion = execSync('google-chrome --product-version').toString();
-var browser = 'Chrome ' + chromeVersion.split('.').slice(0, 2).join('.');
 var path = require('path');
+var Promise = require('rsvp').Promise;
```

Comment:
do we need `rsvp` here? Node 6 has native Promise support which should be enough in most cases 🤔

Reply:
Removed.